packages/terra-demographics-banner/lib/DemographicsBanner.scss

```diff
@@ -0,0 +1,94 @@
+@import '~terra-mixins';
+@import './variables';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: 767px) {
+    display: block;
+  }
+}
+
+.terra-DemographicsBanner--value {
```

Comment:
Most of these classes are double dashed, which I thought represented modifiers. Should these all use a single dash unless they are modifiers? 
```
.terra-DemographicsBanner-value
```